packages/next/src/seo/root/site-metadata/types.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Represents the Site metadata of a site.
+ */
+export interface SiteMetadata {
+	siteTitle?: string;
+	description?: string;
+	locale?: string;
```

Comment:
Here and everywhere, if we don't have a reason to treat an explicit undefined differently than an implicit on, we should add `| undefined` to any `?:` defs